### PR TITLE
Config map pattern

### PR DESCRIPTION
**Pull request by goruha**

### CODE REVIEW

**Comment by osterman** [comment]
Rebase

File: incubator/monochart/templates/configmap.yaml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-{{ include "common.labels.standard" . | indent 4 }}
-data:
-{{- with .Values.configMap.files }}
+{{- with $config.files }}
 {{ toYaml . | indent 2 }}
 {{- end }}
 
+{{- range $key, $value := $config.files_yaml }}
+  {{ $key }}: |
+{{ regexReplaceAll "(\\s+\\w+:\\s+)\\\\(\\!\\w+)" ( toYaml $value ) "${1}${2}" | indent 4 }}
```
Comment (osterman) [inline incubator/monochart/templates/configmap.yaml]: Can we add a comment in the go template what this is doing?

File: incubator/monochart/values.yaml [hunk 2]
```diff
@@ -8,13 +8,20 @@ image:
...
+configMaps:
+  # default:
+  #   labels:
+  #     name: value
+  #   env:
+  #     ENV_NAME: ENV_VALUE
+  #   files:
+  #     "/etc/ssl/test.crt": |-
+  #         ...
+  #   files_yaml:
```
Comment (osterman) [inline incubator/monochart/values.yaml]: Use standard camelCase used elsewhere.

Comment (osterman) [reply incubator/monochart/values.yaml]: (also, I would be okay with calling it just `yaml`, but `filesYaml` also okay)

Comment (osterman) [reply incubator/monochart/values.yaml]: What if we just called it `files` and you detect if it's a string or map?

**Comment by alebabai** [comment]
Should be replaced by this one https://github.com/cloudposse/charts/pull/161

**Pull request closed without merging**